WordPress/src/main/java/org/wordpress/android/ui/jetpackoverlay/JetpackFeatureRemovalBrandingUtil.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.ui.jetpackoverlay
+
+import org.wordpress.android.ui.jetpackoverlay.JetpackFeatureRemovalPhase.PhaseFour
+import org.wordpress.android.ui.jetpackoverlay.JetpackFeatureRemovalPhase.PhaseOne
+import org.wordpress.android.ui.jetpackoverlay.JetpackFeatureRemovalPhase.PhaseThree
+import org.wordpress.android.ui.jetpackoverlay.JetpackFeatureRemovalPhase.PhaseTwo
+import org.wordpress.android.util.BuildConfigWrapper
+import javax.inject.Inject
+
+class JetpackFeatureRemovalBrandingUtil @Inject constructor(
+    private val buildConfigWrapper: BuildConfigWrapper,
+    private val jetpackFeatureRemovalPhaseHelper: JetpackFeatureRemovalPhaseHelper,
+) {
+    fun shouldShowPhaseOneBranding(): Boolean {
+        if (buildConfigWrapper.isJetpackApp) return false
```

Comment:
❓  Is the check `if (buildConfigWrapper.isJetpackApp) return false` necessary as `JetpackBrandingUtils.shouldShowJetpackBranding()` already has this check?

OR alternatively can we omit the usage of the class `JetpackBrandingUtils` and use the below function in `JetpackFeatureRemovalBrandingUtil`?. That way, we can call the `JetpackFeatureRemovalBrandingUtil` from Scan screens to check whether the branding needs to be displayed.



```
fun shouldShowJetpackBranding(): Boolean {
        return isWpComSite() && jetpackPoweredFeatureConfig.isEnabled() && !buildConfigWrapper.isJetpackApp
    }
```
Let me know your thoughts 🤔

Reply:
> ❓ Is the check if (buildConfigWrapper.isJetpackApp) return false necessary as JetpackBrandingUtils.shouldShowJetpackBranding() already has this check?

Thank you for raising the question @AjeshRPai 🙇 
I agree the check is redundant given it's already covered by `JetpackBrandingUtils`. I will remove it and adapt the unit tests 👍 

> OR alternatively can we omit the usage of the class `JetpackBrandingUtils` and use the below function in `JetpackFeatureRemovalBrandingUtil`?. That way, we can call the `JetpackFeatureRemovalBrandingUtil` from Scan screens to check whether the branding needs to be displayed.
> [...]
> Let me know your thoughts 🤔

My initial idea was to rely only on `JetpackBrandingUtils` in the UI layer since it's also used to update the visibility of the banner when scrolling and it also already had the logic for the additional checks. IIRC porting these checks to `JetpackFeatureRemovalBrandingUtil` would mean reimplementing them there which is something I've tried to avoid in the first place.
If you're ok with it my thoughts are to keep the current logic as is, without the redundant check for Jetpack app.

Reply:
Hey @ovitrif 

>IIRC porting these checks to JetpackFeatureRemovalBrandingUtil would mean reimplementing them there which is something I've tried to avoid in the first place.

Oh, I see. Then it makes sense to keep the current logic as is after removing the redundant check 👍🏼